composer/loggers/progress_bar_logger.py

```diff
@@ -163,18 +168,9 @@ def __init__(
 
     @property
     def _current_pbar(self) -> Optional[_ProgressBar]:
-        if self.is_train is None:
-            return None
-        return self.train_pbar if self.is_train else self.eval_pbar
-
-    @_current_pbar.setter
-    def _current_pbar(self, pbar: Optional[_ProgressBar]):
-        assert self.is_train is not None, 'Cannot set pbar if self.is_train is not set.'
-
-        if self.is_train:
-            self.train_pbar = pbar
-        else:
-            self.eval_pbar = pbar
+        if self.eval_pbar is not None:
```

Comment:
if we remove `self.is_train`, then we should also just remove `_current_pbar` and have the code be explicit about which bar they are using.